code/modules/vending/games.dm

```diff
@@ -20,7 +20,8 @@
 					/obj/item/skillchip/bonsai=2,
 					/obj/item/skillchip/wine_taster=2,
 					/obj/item/skillchip/light_remover=2,
-					/obj/item/skillchip/useless_adapter=5)
+					/obj/item/skillchip/useless_adapter=5,
+					/obj/item/dyespray=3)
```

Comment:
Just a bit of cleanup while you're here, change this to a slightly better format like this:
```
listvar = list(
  foo,
  bar,
)
```